splinterd/Cargo.toml

```diff
@@ -66,9 +66,10 @@ experimental = [
     "service-arg-validation"
 ]
 
-biome = ["splinter/biome", "database"]
-biome-credentials = ["splinter/biome-credentials", "json-web-tokens", "biome"]
-biome-key-management = ["splinter/biome-key-management", "json-web-tokens", "biome"]
+biome = ["splinter/biome",  "database"]
+biome-credentials = ["splinter/biome-credentials", "biome-refresh-tokens", "json-web-tokens", "biome"]
```

Comment:
It seems like biome-key-management and biome-credentials shouldn't depend on the biome-refresh-tokens feature, since there are no splinterd code changes here and the dependencies should be handled inside libsplinter itself.

Reply:
Maybe same with json-web-tokens?